activiti-cloud-services-runtime-bundle/activiti-cloud-services-security-policies/src/main/java/org/activiti/cloud/services/SecurityPolicyService.java

```diff
@@ -0,0 +1,98 @@
+package org.activiti.cloud.services;
+
+import org.activiti.conf.SecurityProperties;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+@Component
+public class SecurityPolicyService {
+
+    private SecurityProperties securityProperties;
+
+    @Autowired
+    public SecurityPolicyService(SecurityProperties securityProperties){
+        this.securityProperties = securityProperties;
+    }
+
+    public boolean policiesDefined(){
```

Comment:
@ryandawsonuk I will totally delegate this one to you, but I'm struggling with PolicyService and PoliciesService. Is it a service that deals with a bunch of Policies right? Should we call it in plural or does it make sense to keep it in singular? When I see the Enum SecurityPolicy it make sense to be singular, but for the service if the user can define multiple policies I would just be tempted to use plural. Completely up to you.

Reply:
Yeah I like the plural actually - I'll change it